tanner/config.py

```diff
@@ -18,9 +18,11 @@
                    'REDIS': {'host': 'localhost', 'port': 6379, 'poolsize': 80, 'timeout': 1},
                    'EMULATORS': {'root_dir': '/opt/tanner'},
                    'EMULATOR_ENABLED': {'sqli': True, 'rfi': True, 'lfi': True, 'xss': True, 'cmd_exec': True,
-                                        'php_code_injection': True, 'php_object_injection': True, "crlf": True},
+                                        'php_code_injection': True, 'php_object_injection': True, "crlf": True,
+                                        "xxe_injection": True},
                    'SQLI': {'type': 'SQLITE', 'db_name': 'tanner_db', 'host': 'localhost', 'user': 'root',
                             'password': 'user_pass'},
+                   'OUT_OF_BAND': {'xxe_injection': False},
```

Comment:
`'xxe_injection': {'OUT_OF_BAND': False}` is more suitable.

Reply:
Okay